src/main/java/picocli/CommandLine.java

```diff
@@ -2036,7 +2035,6 @@ public int execute(String... args) {
             }
         } catch (ExecutionException ex) {
             try {
-                @SuppressWarnings("unchecked")
                 Exception cause = ex.getCause() instanceof Exception ? (Exception) ex.getCause() : ex;
```

Comment:
`Exception.getCause` returns a `Throwable`. We never catch Throwable so we can safely downcast it to Exception, but I believe we do need to keep this annotation to suppress the compiler warning.